internal/api/v1/application/part.go

```diff
@@ -74,9 +74,9 @@ func GetPart(c *gin.Context) apierror.APIErrors {
 		return apierror.AppIsNotKnown(appName)
 	}
 
-	if app.Workload == nil {
-		// While the app exists it has no workload, and therefore no chart to
-		// export
+	if partName != "manifest" && app.Workload == nil {
+		// While the app exists it has no workload, and therefore no chart/image/values to
+		// export. Manifest however will be fine.
```

Comment:
I prefer to early return, because it has less cognitive complexity:

```golang
if partName == "manifest" {
	return fetchAppManifest(c, app)
}

if app.Workload == nil {
	return apierror.NewBadRequestError("no chart available for application without workload")
}
```

Reply:
Ah, make it separate from the main switch. doable.

Reply:
Done, commit 281c55964128a18f8a6bcf3b4d75b06e80395032